test/e2e/parameters/keys.dart

```diff
@@ -197,6 +199,7 @@ enum WidgetKey {
   VerifyPhone,
   WelcomeMessage,
   WelcomeMessageField,
+  BigAvatarProfileField,
```

Comment:
Should be alphabetically sorted.

Reply:
That's right, i'll correct it

Reply:
fixed on 3bdb4c364a9c19145ac3970c4389cdc034f8f30f commit